.terraform.lock.hcl

```diff
@@ -2,24 +2,24 @@
 # Manual edits may be lost in future updates.
 
 provider "registry.terraform.io/equinix/equinix" {
```

Comment:
For reusable modules (pretty much anything that we expect will be run by someone else), we've been moving towards removing & gitignore-ing lock files in favor of open version constraints